packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -12,10 +12,12 @@
 - Fixed failing checkout tests [#2195](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2195)
 - [BUG] Fixed "getCheckboxProps is not a function" when rendering checkout page in generated app.[#2140](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2140)
 - [BUG] Fix images being fetced multiple times on Safari [#2223](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2223)
+- Support Node 22 [#2218](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2218)
```

Comment:
No need for this duplicate line. See line 3 above. I rearranged the order of items so they're in the correct groups (bug fixes, features, etc)

Reply:
Oh one more thing: please delete line 7, which mentions the page meta tags.